geozero/src/wkb/wkb_reader.rs

```diff
@@ -293,17 +424,59 @@ pub(crate) fn process_wkb_geom_n<R: Read, P: GeomProcessor>(
     }
 }
 
+fn emit_coord<P: GeomProcessor>(
```

Comment:
I don't think this is a good approach here - you first create coordinate tuple on line 462, and then you discover you don't even need it if multi_dim is false.  I would inline this function

Reply:
Fixed with latest commit